packages/embark-ui/src/components/ContractOverview.js

```diff
@@ -171,12 +186,27 @@ class ContractFunction extends Component {
                : (ContractFunction.isPureCall(this.props.method) &&
                    this.makeBadge('success', 'white', 'call')) ||
                    this.makeBadge('warning', 'black', 'send')}
+              {ContractFunction.isPayable(this.props.method) &&
+                this.makeBadge('light', 'black', 'payable')
+              }
             </div>
           </CardTitle>
         </CardHeader>
         {!ContractFunction.isFallback(this.props.method) &&
          <Collapse isOpen={this.state.functionCollapse} className="relative">
           <CardBody>
+            {ContractFunction.isPayable(this.props.method) &&
+              <Form inline>
+                <Label for={this.props.method.name + '-value'} className="mr-2 font-weight-bold contract-function-input">
```

Comment:
This is a form input like all the other inputs for method parameters. Since it's possible for Smart Contract authors to call their parameters indeed `"value"`, we need to ensure that there's no name conflict. 

The above ensures a unique name for any individual method's `value` input.